x-pack/test/security_solution_cypress/cypress/e2e/explore/guided_onboarding/tour.cy.ts

```diff
@@ -30,12 +29,12 @@ import { visit } from '../../../tasks/navigation';
 import { startAlertsCasesTour } from '../../../tasks/api_calls/tour';
 import { deleteAlertsAndRules } from '../../../tasks/api_calls/common';
 
-describe('Guided onboarding tour', { tags: ['@ess'] }, () => {
+// re-enable this when https://github.com/elastic/kibana/pull/181442 is merged
+describe.skip('Guided onboarding tour', { tags: ['@ess'] }, () => {
```

Comment:
Is this test failing?

Reply:
no, I had skipped it because my branch was opened at the time it was failing and you were working on the fix. It's unskipped in my local, and working!
I'm fixing a couple of other comments then will push a commit. No changes to this file anymore!

Reply:
@angorayc the change was reverted in this last commit https://github.com/elastic/kibana/pull/184169/commits/7f601451a5169cdb9b502089502369f426c12b5a!